ci/prow/boskos/create_projects.sh

```diff
@@ -16,18 +16,25 @@
 
 set -e
 
-readonly FIRST=${1:?"First argument is the first number of the new project(s)."}
-readonly NUMBER=${2:?"Second argument is the number of new projects."}
-readonly BILLING_ACCOUNT=${3:?"Third argument must be the billing account."}
-readonly OUTPUT_FILE=${4:?"Fourth argument should be a file name all project names will be appended to in a resources.yaml format."}
+readonly NUMBER=${1:?"First argument is the number of new projects to create."}
+readonly BILLING_ACCOUNT=${2:?"Second argument must be the billing account."}
 
+readonly RESOURCE_FILE="resources.yaml"
+
+start=$(grep -n "knative-boskos-" ${RESOURCE_FILE} | wc -l)
```

Comment:
1. To be safe, ensure "resources.yaml" exists and is writable.
1. Add a single comment here saying you're fetching the index of the next boskos project from the resources file. 
1. `start` should be `START`.

Reply:
Ditto